improve abstract self-consistency (#227)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

--- abstract

In network protocol exchanges it is often the case that

> one entity requires believeable evidence about the operational state of a remote

Reviewer: ```suggestion
one entity requires believable evidence about the operational state of a remote
```

peer.  Such evidence is typically conveyed as claims
about the peer's software and hardware platform, and is subsequently appraised in order
to assess the peer's trustworthiness.  The process of generating and

> appraising this kind of evidence goes under the name of remote

Reviewer: ```suggestion
appraising this kind of evidence is known as remote
```
